common/archiver/README.md

```diff
@@ -45,13 +46,22 @@ type HistoryArchiver interface {
 
 ```go
 type VisibilityArchiver interface {
+    // Archive is used to archive one workflow visibility record. Check the Archive() method of the HistoryArchiver interface for parameters' meaning and requirements for this method. 
```

Comment:
Does go doc support link to code?

Reply:
Oh, I am referring to the method and its comment starting from L26 in this file, not the actual code. I will change the wording. Sorry for the confusion.